README.md

```diff
@@ -30,30 +30,44 @@ Note: This bundle is also conjure-up enabled. Refer to the
 
 Any of the services provided can be scaled out post-deployment. The charms
 update the status messages with progress, so it is recommended to run
-`watch juju status` to monitor the cluster coming up. After it is deployed you
-need to grab the kubectl and configuration from the Kubernetes leader node to
-control the cluster:
+`watch juju status` to monitor the cluster coming up.
 
-    juju scp kubernetes/0:kubectl_package.tar .
-    tar zxf kubectl_package.tar
-    ./kubectl get pods
+After the cluster is deployed you need to grab the kubectl binary and
+configuration for your cluster from the Kubernetes **master unit** to control
+the cluster. To find the master unit check the `juju status` output or run
+a command on all kubernetes units to detect the leader:  
+
+    juju run --service kubernetes is-leader
```

Comment:
@chuckbutler Yes we did. The master unit will prefix all status messages with "(master)".  @battlemidget had some questions about kubernetes leader/master today so I decided to put more words about finding the leader/master. Do you disagree with anything I wrote here?